x-pack/plugin/transform/src/main/java/org/elasticsearch/xpack/transform/TransformExtension.java

```diff
@@ -20,4 +21,6 @@ public interface TransformExtension {
      * source settings.
      */
     Settings getTransformDestinationIndexSettings();
+
+    TimeValue getMinFrequency();
```

Comment:
Temporarily you'll have to implement a default for this method, otherwise anyone who needs to update the submodule in the serverless repo will find the serverless transform extension won't compile.

Creating changes that span the public repo and serverless at the moment is a bit of a pain. The way to do it that causes the least disruption to others is:

1. Implement a change in the public repo that is compatible with the current serverless code
2. Implement a change in the serverless repo that makes serverless use the new functionality in the ideal way
3. (Not always necessary) Implement another change in the public repo that removes any workarounds needed in step 1

Reply:
Yes, that makes sense. Should have thought of that myself...

Reply:
fixed